### DISCUSSION TITLE
tinyca - Permission denied creating first localhost certs

### DISCUSSION THREAD

**Discussion by jswingen** [smallstep/certificates#559] [Q&A] [answered]
I am building a tinyca and get the following error when trying to generate a localhost cert using the 

step ca certificate "localhost" localhost.crt localhost.key        command

✔ Provisioner: xxxxxxx@xxxxxx.com (JWK) [kid: xgHsl48wBOhdhnorlO-UA0WEbkLpNxC7_1d5WcosfRM]
✔ Please enter the password to decrypt the provisioner key:
✔ CA: https://tinyca.xxxxx.local
open localhost.crt: permission denied

Any thoughts.  Thank you.

**Comment by maraino** [comment, marked as answer]
@jswingen it looks like the error is because you're in a directory without permissions or there's an existing localhost.crt that you cannot overwrite.

**Reply by jswingen** [reply]
Great input Maraino, thanks.  Will give it a try.

**Reply by jswingen** [reply]
Elevated the CA process terminal login (sudo -s), restarted the CA, and the localhost cert generated.  Thanks for the input.
